New package: CLASS v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: CLASS
- Repository: https://github.com/hersle/CLASS.jl
- Created by: @hersle
- Version: v0.1.0
- Commit: 6d139e1460f2fe18a705dea7ef2faacbae94948b
- Reviewed by: @hersle
- Reference: https://github.com/hersle/CLASS.jl/issues/1#issuecomment-3013170716
- Description: A simple Julia wrapper for the cosmological Boltzmann solver CLASS that reads its output as DataFrames.
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835986eb1d7f9bb366e90f896752668e52c7c75926fe4dd39d0fc81de3f4b357dbae31469cb14c7ce40b2c2baf41b986730ba37feaee49ae637ac2ed9a6f2923c3bfa98737eccf4f2f7ebf43a3ea698ae2fd9693304ea390c384aa292e9c2848ee8910b5c2bacf6f1b5b7684ecef24dbfce3c -->

hersle commented:
Close

goerz commented:
[noblock] The name `CLASS` should be fine, see https://github.com/JuliaRegistries/General/pull/133795#issuecomment-3012616441

JuliaRegistrator committed:
New package: CLASS v0.1.0

UUID: a61081fd-7c4d-44c2-8709-3eabc28bd84c
Repo: https://github.com/hersle/CLASS.jl.git
Tree: e6b51750d3cc1b2aa2afc717b21fe1d0dbc08a51

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3

hersle commented:
[noblock] Is merging still blocked? Due to

> Name does not meet all of the following: starts with an upper-case letter, ASCII alphanumerics only, **not all letters are upper-case**.

Maybe @goerz / @GunnarFarneback

goerz commented:
[noblock] Yeah, it still needs a manual merge. Unfortunately, I don't have sufficient permissions to merge manually. The standard procedure is to post on the `#pkg-registration` channel on Slack towards the end of the 3-day waiting period.

(I think we should get rid of rule against all-caps in the bot; the minimum-length requirement is much more effective at keeping out acronyms, but that's a separate thing to look into at some point)

hersle commented:
[noblock] I need help with a manual merge here. CLASS.jl is a wrapper around [CLASS](http://class-code.net/), which is a popular C code in cosmology for solving the linearized Einstein-Boltzmann equations. CLASS is short for Cosmic Linear Anisotropy Solving System. According to [package naming convention #6](https://pkgdocs.julialang.org/v1/creating-packages/#Package-naming-rules). I have named the wrapper after the code. A manual merge is needed because CLASS only contains capital letters. Posting this on Slack now.
